app/code/Magento/Customer/Ui/Component/DataProvider/Document.php

```diff
@@ -93,7 +94,12 @@ public function __construct(
     }
 
     /**
-     * @inheritdoc
```

Comment:
Please revert the comment and just remove curly braces `{}`. Please refer to the guide https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#inheritdoc